locate/projects/locate/src/lib/components/locate.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, OnInit } from "@angular/core";
+import { ApiService } from "../services/api.service";
+
+@Component({
+    selector: "lib-locate",
+    templateUrl: "./locate.component.html",
+    styleUrls: ["./locate.component.css"],
+})
+export class locateComponent implements OnInit {
+    constructor(private API: ApiService) {}
+
+    userInput: string = "";
```

Comment:
Just FYI, you don't need to declare the type with `:string` if you're setting the value to "". It is inferred to be a string from that.